test/unit/test_str.c

```diff
@@ -228,7 +228,7 @@ bool test_rz_str_split_list(void) {
 	rz_list_free(l2);
 
 	char s3[] = "Hello  World\tAnd \t Everyone\t";
-	RzList *l3 = rz_str_split_list_regex(s3, "[[:blank:]]+", 0);
+	RzList *l3 = rz_str_split_list_regex(s3, "\\s+", 0);
```

Comment:
Can we support `\s` too? I find it more intuitive than `[[:blank:]]`

Reply:
I suggest this is done in a separate PR after this one merged.

Reply:
Depends on how it would be implemented. It would be better to not go back and forth on changes unless necessary.

Reply:
Added a test for it. It still works.